src/models/raster/constants.ts

```diff
@@ -36,4 +48,7 @@ export const VALIDATIONS = {
   sensor: {
     pattern: '^(?!\\s).+(?<!\\s)$',
   },
+  polygonPartsEntityName: {
```

Comment:
Please adapt the regex so value cannot end with _
example: atm this is valid: x_4gt_